src/lang.rs

```diff
@@ -1052,8 +1063,58 @@ fn req_ident(input: Span) -> IResult<Span, String> {
     .parse(input)
 }
 
+fn escaped_chars(input: Span) -> IResult<Span, Span> {
+    alt((
+        tag("\\"),
+        tag("n"),
+        tag("n"),
+        tag("t"),
+        tag("0"),
+        tag("'"),
+        tag("\""),
+        take(1usize),
```

Comment:
what does `take(1usize)` do here?

Reply:
It's to allow passing through things like `\d`.  Added a comment.